src/inject.js

```diff
@@ -243,6 +243,12 @@ const processStyleDefinitions = (
     }
 };
 
+// Sum up the lengths of the stringified style definitions (which was saved as _len property)
+// and use modulus to return a single byte hash value.
```

Comment:
This comment would be 💯  if it explained why we are adding the length to the hash.